File: Medical_Profile/providers.cs

namespace Medical_Profile
{
 public class Providers_return
 {
  public string Token { get; set; }
  public string next { get; set; }
  public Provider_return[] providers { get; set; }
  public int totalcount { get; set; }
 }

 public class Provider_return
 {
  public string Firstname { get; set; }
  public string Specialty { get; set; }
  public string Displayname { get; set; }
  public string Lastname { get; set; }
  public int Providerid { get; set; }
  public string Homedepartment { get; set; }
  public string Providerusername { get; set; }
  public string Providertype { get; set; }
 }
}

File: Medical_Profile/providers.vb
Imports System

Public Class Providers_return
  Public Property Token As String
  Public Property [next] As String
 Public Property providers As Provider_return()
 Public Property totalcount As Integer
End Class

Public Class Provider_return
  Public Property Firstname As String
  Public Property Specialty As String
  Public Property Displayname As String
  Public Property Lastname As String
  Public Property Providerid As Integer
  Public Property Homedepartment As String
  Public Property Providerusername As String
  Public Property Providertype As String
End Class
